source/_docs/configuration/templating.markdown

```diff
@@ -1296,3 +1296,31 @@ The default priority of operators is that the filter (`|`) has priority over eve
 {% endraw %}
 
 Would round `10` to 2 decimal places, then divide `states('sensor.temperature')` by `10` (rounded to 2 decimal places so 10.00). This behavior is maybe not the one expected, but priority rules imply that.
+
+### Rate limits for template updates
```

Comment:
This section seems out of place, this document introduces all the different functions.
None of these are rate-limited as-is. However, some rate-limiting happens in the [automation trigger](https://www.home-assistant.io/docs/automation/trigger/#template-trigger) and [template entity)(https://www.home-assistant.io/integrations/template/) updates. Did you mean to add this to those places instead?

Reply:
I wasn't sure where the best place was, to be honest. I'll be happy to add it there. Also, I think there would be some duplication of information in that case. Would that be a problem?

Reply:
They are two different things, so I do not think that will be an issue.